apps/internal/local/server.go

```diff
@@ -167,7 +175,11 @@ func (s *Server) handler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	_, _ = w.Write(okPage)
+	if len(s.optionSuccessPage) > 0 {
```

Comment:
You can refactor a little to remove this condition:
```go
func New(reqState string, port int, successPage []byte, errorPage []byte) (*Server, error) {
	if len(successPage) == 0 {
		successPage = okPage
	}
	serv := &Server{
		// len(successPage) != 0
		successPage: successPage,
	}
```
With that, you can unconditionally `w.Write(s.successPage)`